### ISSUE TITLE
Allow to suppress lines about classpath artifacts

### ISSUE THREAD

**Issue by php-coder** [ipastusi/wiremock-maven-plugin#12]
Plugin produces a lot of output to the console with names of artifacts that are included in the classpath:
```console
$ mvn wiremock:run
[INFO] --- wiremock-maven-plugin:2.13.0:run (default-cli) @ mystamps ---
[INFO] Adding /Users/coder/git/mystamps.git/target/test-classes to wiremock-maven-plugin classpath
[INFO] Adding /Users/coder/git/mystamps.git/target/classes to wiremock-maven-plugin classpath
[INFO] Adding /Users/coder/.m2/repository/com/github/heneke/thymeleaf/thymeleaf-extras-togglz/2.0.1.RELEASE/thymeleaf-extras-togglz-2.0.1.RELEASE.jar to wiremock-maven-plugin classpath
[INFO] Adding /Users/coder/.m2/repository/org/thymeleaf/thymeleaf/3.0.9.RELEASE/thymeleaf-3.0.9.RELEASE.jar to wiremock-maven-plugin classpath
[INFO] Adding /Users/coder/.m2/repository/ognl/ognl/3.1.12/ognl-3.1.12.jar to wiremock-maven-plugin classpath
...
$ mvn wiremock:run | grep -c 'Adding .* to wiremock-maven-plugin classpath'
72
```
Does it really useful information? Could it be suppressed at all? Otherwise, could you introduce an option for that?

Thanks!

**Comment by automatictester** [comment]
Change implemented to log those under DEBUG level, not INFO. New version Available in Maven Central: 2.13.1. If for some reason you would still like to see these log entries, you will need to run Maven process with log level set to DEBUG:
```mvn wiremock:run -Dorg.slf4j.simpleLogger.defaultLogLevel=debug```

**Comment by php-coder** [comment]
Wow! Thank you!!
